README.md

```diff
@@ -4,14 +4,19 @@
 
 [HAL]: https://crates.io/crates/embedded-hal
 
-## Documentation
+Travis Status: [![Build Status](https://travis-ci.org/nrf-rs/nrf-hal.svg?branch=master)](https://travis-ci.org/nrf-rs/nrf-hal)
```

Comment:
nit: the text is not aligned with the badge

Reply:
Hmm. Any ideas how to fix this?

Reply:
You could inline a css style an manipulate the line height. maybe padding/margin works too :) the problem is that the default html behavior is that the text line of origin aligns (which makes sense in most cases). since badges are just text with a background this looks weird now :) but you should be able to write plain hatml in markdown and it will render properly :)

Reply:
Can I propose we merge this, and then think about making it prettier? I'm very keen to get 0.9 out of the door...

Reply:
Or just remove the text, the badge is enough

Reply:
Done